content-scripts/src/modules/options/some_options.js

```diff
@@ -4,7 +4,7 @@ import removeElement from "../utilities/removeElement";
 //import { getStorage } from "../utilities/storage";
 
 // Function to hide shortcuts 
-export const hideShortcuts = (shortcuts) => {
+export const hideShortcuts = (shortcuts) => new Promise((resolve, reject) => {
```

Comment:
Instead of wrapping the function within a promise, you can just mark it as async

Reply:
Done it!